src/components/sidenav/sidenav.js

```diff
@@ -240,7 +241,11 @@ function SidenavDirective($mdMedia, $mdUtil, $mdConstant, $mdTheming, $animate,
         $mdMedia: $mdMedia
       });
     };
-    var backdrop = !angular.isDefined(attr.mdDisableBackdrop) ? $mdUtil.createBackdrop(scope, "_md-sidenav-backdrop md-opaque ng-enter") : undefined;
+
+    // Only create the backdrop if the backdrop isn't disabled.
+    if (!$mdUtil.parseAttributeBoolean(attr.mdDisableBackdrop)) {
```

Comment:
*  Why parse for boolean value, the `md-disable-backdrop` implies a TRUE. 
*  What need do we have to parse the expression value?